dev-tools/ecs-migration.yml

```diff
@@ -433,6 +433,28 @@
   to: user_agent.original
   alias: true
 
+## PostgreSQL module
+
+- from: postgresql.log.timezone
+  to:
+  alias: true
+
+- from: postgresql.log.thread_id
+  to: process.pid
```

Comment:
Yes. Postgresql cannot log thread IDs, it's not an option, it can only log process IDs: [Doc link](https://www.postgresql.org/docs/9.5/runtime-config-logging.html#AEN33411). I think PG forks per connection or something.

Also worth considering was [the previous documentation](https://github.com/elastic/beats/pull/9308/files#diff-c5822b93185ef6e18113596fdce961d3L15) for the `thread_id` field, which read "Process ID" ;-)